src/ui/flutter_app/lib/screens/game_settings/algorithm_modal.dart

```diff
@@ -50,7 +50,7 @@ class _AlgorithmModal extends StatelessWidget {
           RadioListTile(
             title: const Text('MTD(f)'),
             groupValue: algorithm,
-            value: Algorithms.MTD,
+            value: Algorithms.MTDF,
```

Comment:
Algorithms.MTDf

Reply:
bruh
I did change that but a rebase with some merge conflicts must have "reverted" it :upside_down_face: